webpack/plugins/pre-build-plugin.js

```diff
@@ -0,0 +1,27 @@
+const fs = require("fs");
+const path = require("path");
+const debug = require("debug")("CdkWeb:PreBuildPlugin");
+const PrebuildPlugin = require("prebuild-webpack-plugin");
+const { __ROOT } = require("../common");
+
+module.exports = new PrebuildPlugin({
+  build: (compiler, compilation, matchedFiles) => {
+    debug("copying esbuild to dist");
+    const esbuildRoot = path.resolve(require.resolve("esbuild-wasm"), "../..");
+    fs.copyFileSync(path.resolve(esbuildRoot, "esbuild.wasm"), path.resolve(__ROOT, "dist/esbuild.wasm"));
```

Comment:
up there somewhere I left a comment about this. we can have webpack to do this proper for us.

- since `esbuild.wasm` is not really part of the bundle, it does not need to be copied here. it can be moved to post build plugin.
- `go-wasm.js` is used by sources so we can just alias `go-wasm`: `path.resolve(esbuildRoot, "wasm_exec.js")` and then in sources you can just do `require("go-wasm")`.

if you do the above ^, you will not need a pre build plugin :)